src/app/connector-editor/extension/connector-creator.extension.ts

```diff
@@ -29,6 +29,7 @@ export function createConnectorCreator(): ModelCreator {
             title: 'APP.PROJECT.CONNECTOR_DIALOG.TITLE_CREATE',
             nameField: 'APP.PROJECT.CONNECTOR_DIALOG.CONNECTOR_NAME',
             descriptionField: 'APP.PROJECT.CONNECTOR_DIALOG.CONNECTOR_DESC',
+            allowedCharacters: '^[a-z0-9]{0,26}$',
```

Comment:
Please extract this information to ama-sdk, next to `MODEL_NAME_CHARACTERS` to support its usage from the alfresco repo. Something like `CONNECTOR_NAME_REGEX`. By the way, why do we use a regex string instead of a real Regex object?
`/^[a-z0-9]{0,26}$/`